CleverPush/CleverPush/Source/CPAppBanner.m

```diff
@@ -11,6 +11,10 @@ - (id)initWithJson:(NSDictionary*)json {
         self.name = [json objectForKey:@"name"];
         self.HTMLContent = [json objectForKey:@"content"];
         self.contentType = [json objectForKey:@"contentType"];
+        self.appVersionFilterRelation = [json objectForKey:@"appVersionFilterRelation"];
+        self.appVersionFilterValue = [json objectForKey:@"appVersionFilterValue"];
+        self.fromVersion = [json objectForKey:@"fromVersion"];
```

Comment:
StringForKey ==> ValueForKey will return string value and objectForKey is for the anyObject type so I have use valueForKey because there is no function in NSDictionary class that is call StringForKey